includes/class-amp-theme-support.php

```diff
@@ -1497,6 +1497,38 @@ protected static function has_dependency( WP_Dependencies $dependencies, $curren
 		return false;
 	}
 
+	/**
+	 * Check if a handle is exclusively a dependency of another handle.
+	 *
+	 * For example, check if dashicons is being added exclusively because it is a dependency of admin-bar, as opposed
+	 * to being added because it was directly enqueued by a theme or a dependency of some other style.
+	 *
+	 * @since 1.4.2
+	 *
+	 * @param WP_Dependencies $dependencies         Dependencies.
+	 * @param string          $handle               Handle.
+	 * @param string          $exclusive_dependency Exclusive dependency handle.
```

Comment:
If I understand the logic correctly, then `$exclusive_dependency` is the wrong name for the third argument here. The dependency that is checked on its exclusivity is `$handle`.